### PR TITLE
Add further details/workaround for #940

### PR DESCRIPTION
**Pull request by russelldb**
This is for discussion/collaboration. After discussion in slack we agreed to ship 2.2.5 with this defect, and to call attention to it in the release notes and docs.

- is this enough attention?
- is the workaround correct?

### CODE REVIEW

**Comment by martinsumner** [comment]
Something up with Markdown on headers in the earlier release notes section.  Missing space?

**Comment by martinsumner** [comment]
On the specific issue of providing enough attention in the release notes + 1

**Comment by russelldb** [comment]
Added the spaces, 'cos yeah, weird.

**Comment by martinsumner** [comment]
I think the release contents need a separate header - so they don't appear to be under known issues

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -24,9 +24,50 @@ replication. The rest of the changes are fixes (riak-core claim,
...
+   {rt_heartbeat_timeout, 15},
+   {fullsync_use_background_manager, true}
+  ]},
+```
+
+Read more about configuring
+[MDC](http://docs.basho.com/riak/kv/2.2.3/configuring/v3-multi-datacenter/)
+replication.
+
+More details about the issue can be found
```
Comment (ghost) [inline RELEASE-NOTES.md]: @russelldb - you've already provided the useful config directly, no point in directing users to the comment again, also let's make it consistent with the format you've used for others.

```
More details about the issue can be found in riak_repl/782 [2.2.5 - [enoent] - riak_repl couldn't create log dir "data/riak_repl/logs" ](https://github.com/basho/riak/issues/940)
```

**Comment by russelldb** [comment]
Pushed to address all current comments on this issue…

**Comment by ghost** [comment]
+1 

squirrel

**Pull request merged**